### ISSUE TITLE
Add CI targets for tarantool 1.9, 1.10 and 2.0

### ISSUE THREAD

**Issue by Totktonada** [tarantool/graphql.0#205]
Need to test on tarantool 1.9, 1.10 and 2.0 (head of the branches). It is worth to do, because we'll use `is_async` net.box flag conditionally in `accessor_shard_cache` after #98.

**Comment by Totktonada** [comment]
I'm going to archive the repository. I'll proceed as follows:

* Mark all open pull requests with the `archived` label and close.
* Mark all open issues with `archived` and `wontfix` labels and close.
* Archive the repository.

Consider the following projects:

* https://github.com/tarantool/graphql
* https://github.com/tarantool/graphqlapi
* https://github.com/tarantool/graphqlide
